clients/python-wrapper/lakefs/client.py

```diff
@@ -106,6 +110,70 @@ def version(self) -> str:
         return self._server_conf.version
 
 
+def get_identity_token(session: 'boto3.Session') -> str:
```

Comment:
If it's not intended to be called directly by users:
 ```suggestion
def _get_identity_token(session: 'boto3.Session') -> str:
```